apps/client/src/app/page.tsx

```diff
@@ -13,52 +14,50 @@ export default function Home() {
   });
 
   return (
-    <div className='mx-auto flex h-screen w-screen flex-col items-center'>
-      {session ? (
-        <div className='flex flex-col items-center'>
-          <UserProfile user={session?.user} />
-          <Link
-            href='/api/auth/signout?callbackUrl=/'
-            className='text-3xl text-violet-500'
-          >
-            Sign Out
-          </Link>
-        </div>
-      ) : (
-        <div className='flex flex-col items-center'>
-          <h1 className='text-5xl'>Not logged In</h1>
-          <Link href='/api/auth/signin' className='text-3xl text-red-500'>
-            Sign In
-          </Link>
-        </div>
-      )}
-      {/* <Image 
-        src= "/images/starBg.jpg"
-        alt='background'
-        layout = 'fill'
-        /> */}
-      <Link href='myAccount' className='text-2xl text-green-600'>
-        My Account
-      </Link>
-      {!mode ? (
-        <div className='flex h-screen flex-row items-end'>
-          <button
-            className='my-64 rounded-lg border border-gray-500 bg-gray-200 px-8 py-2 text-4xl text-sky-600'
-            onClick={() => setMode(true)}
-          >
-            Play
-          </button>
-        </div>
-      ) : (
-        <div>
-          <button
-            className='my-64 rounded-lg border border-gray-500 bg-gray-200 px-8 py-2 text-4xl text-sky-600'
-            onClick={() => setMode(false)}
-          >
-            2 Player
-          </button>
-        </div>
-      )}
+    <div>
+      <NavBar session={session} />
+      <div className='mx-auto flex h-screen w-screen flex-col items-center'>
+        {session ? (
+          <div className='flex flex-col items-center'>
+            <UserProfile user={session?.user} />
+            <Link
+              href='/api/auth/signout?callbackUrl=/'
+              className='text-3xl text-violet-500'
+            >
+              Sign Out
+            </Link>
+          </div>
+        ) : (
+          <div className='flex flex-col items-center'>
+            <h1 className='text-5xl'>Not logged In</h1>
+            <Link href='/api/auth/signin' className='text-3xl text-red-500'>
+              Sign In
+            </Link>
+          </div>
+        )}
+        <Link href='myAccount' className='text-2xl text-green-600'>
+          My Account
+        </Link>
+        {!mode ? (
+          <div className='flex h-screen flex-row items-end'>
```

Comment:
This bad boy `h-screen` make the home page's height behave in weird manner
<img width="1470" alt="Screenshot 2567-10-12 at 10 29 58" src="https://github.com/user-attachments/assets/71ee100c-3065-40a1-94ba-892d7198ef85">

Reply:
i think in gino's work he has fixed this already so i didn't deal with it i will just delete it and wait for gino

Reply:
I change it to  an OK one.